test/system/410-selinux.bats

```diff
@@ -63,4 +63,24 @@ function check_label() {
     check_label "--security-opt label=level:s0:c1,c2" "container_t" "s0:c1,c2"
 }
 
+# pr #6752
+@test "podman selinux: inspect multiple labels" {
+    if [ ! -e /usr/sbin/selinuxenabled ] || ! /usr/sbin/selinuxenabled; then
+        skip "selinux disabled or not available"
+    fi
+
+    run_podman run -d --name myc \
+               --security-opt seccomp=unconfined \
+               --security-opt label=type:spc_t \
+               --security-opt label=level:s0 \
+               $IMAGE sh -c 'while test ! -e /stop; do sleep 0.1; done'
+    run_podman inspect --format='{{ .HostConfig.SecurityOpt }}' myc
+    is "$output" ".*label=type:spc_t.*"   "'podman inspect' includes type"
+    is "$output" ".*label=level:s0.*"     "'podman inspect' includes level"
+    is "$output" ".*seccomp=unconfined.*" "'podman inspect' includes seccomp"
```

Comment:
Does anyone know if the output of `inspect` is guaranteed to be consistent? It'd be nice to do this in one single line.

Reply:
In ordering?

Reply:
Yes. In lots of iterations, both of `podman inspect` and recreating the container, the only value I've seen is:
```
[label=type:spc_t,label=level:s0 seccomp=unconfined]
```
If I change the order of `level` and `type`, the `inspect` order reflects command-line order. `seccomp` is always last regardless of command-line order.

...but that doesn't prove anything; it could just be that I've been lucky so far.

Reply:
From the way the code is currently written, the order should never change, though that's not really an API guarantee

Reply:
That's good enough for me - we don't need an API guarantee, since the tests are tightly bound: any change to the code will be caught early. I've pushed a change that tests in one go. Thank you for checking!